project_app/.eslintrc.js

```diff
@@ -0,0 +1,76 @@
+module.exports = {
+    root: true,
+    parserOptions: {
+        ecmaVersion: 2020,
+        sourceType: "module",
+        ecmaFeatures: {
+            jsx: true,
+        },
+    },
+
+    env: {
+        browser: true,
+        node: true,
+        es6: true,
+    },
+
+    settings: {
+        react: {
+            version: "detect",
+        },
+        "import/resolver": {
+            node: {
+                extensions: [".ts", ".tsx"],
+            },
+        },
+    },
+
+    plugins: ["@typescript-eslint"],
+    extends: [
+        "plugin:react/recommended",
+        "google",
+        "prettier",
+        "next/core-web-vitals",
+        "plugin:@typescript-eslint/recommended",
+        "airbnb",
+        "prettier",
+        "plugin:jsx-a11y/recommended",
+        "plugin:prettier/recommended",
+        "plugin:sonarjs/recommended",
+        "plugin:security/recommended",
+        "plugin:react-hooks/recommended",
```

Comment:
this is a lot of plugins 🤔 what do you think? @enricodelarosa

Reply:
I have some intsalled twice - I will remove the duplication